.github/workflows/pull-request.yml

```
@@ -272,6 +280,126 @@ jobs:
+          set -e
+          echo "Building Docker image..."
+          docker build -t talawa-admin-app .
+          echo "Docker image built successfully"
+      - name: Run Docker Container
+        run: |
+          set -e
+          echo "Started Docker container..."
+          docker run -d --name talawa-admin-app-container -p 4321:4321 talawa-admin-app
+          echo "Docker container started successfully"
+      - name: Check if Talawa Admin App is running
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:35: Double quote to prevent globbing and word splitting

(shellcheck)